src/content/reference/domain-allowlist/index.md

```diff
@@ -18,30 +18,56 @@ user_questions:
 Below is a list of the external domains we require access to for our clusters to function.
 
 - alpinelinux.org
+  - domains:
     - `*.alpinelinux.org`
+  - Alpine container images may update their package index.
 - amazonaws.com
+  - domains:
     - `*.amazonaws.com`
+  - AWS services are used for a variety of tasks, such as etcd backup storage.
 - azurecr.io
+  - domains:
     - `giantswarm.azurecr.io`
+  - Container images are hosted on Azure Container Registry.
 - cloudfront.net
+  - domains:
     - `*.cloudfront.net`
-- coreos.com
-    - `*.coreos.com`
+  - Operators may pull from sites behind Cloudfront.
 - docker.com
+  - domains:
     - `*.docker.com`
+  - Container images are hosted on Dockerhub.
```

Comment:
i'm not too sure why we need access to docker.com - the registry is hosted at docker.io. i don't want to remove it unnecessarily though.